dashboard/app/views/pd/session_attendance/no_enrollment_match.haml

```diff
@@ -3,5 +3,5 @@
   We don't recognize your email address in our roster. In order to mark attendance you must be signed into an account
   that is associated with your workshop enrollment. If you have not enrolled in the workshop yet
   = " "
-  %a{href: CDO.studio_url("/pd/workshops/#{@session.workshop.id}/enroll"), target: "_blank"}> click here to enroll
+  %a{href: CDO.studio_url("/pd/workshops/#{@session.workshop.id}/join"), target: "_blank"}> click here to enroll
```

Comment:
I updated this URL to go to the Join Workshop page instead since this notice relates to a user trying to mark their attendance for a workshop (meaning the user likely already is aware of the workshop and doesn't need a full marketing page's worth of information, they just need the enroll button).